document/CoreFormat.mdk

```diff
@@ -340,9 +340,51 @@ run: {
 ```
 ~
 
+### `logicalLocations` property { #run-logicalLocations }
+
+A `run` object should contain a property named `logicalLocations` whose value is a dictionary,
+each of whose properties represents the logical location of one or more results produced in the course of the run.
+
+With one exception described in &sect;[#location-logicalLocationKey], each property name in the `logicalLocations` object shall be
+a string representing the logical location where the result was produced, in a format consistent with
+the programming language in which the programmatic construct specified by that logical location was expressed.
+We refer to this string as a "fully qualified logical name".
+See &sect;[#location-fullyQualifiedLogicalName] for examples.
```

Comment:
The string I'm referring to here is the property name on the `logicalLocations` object ("With one exception ... each property name ... shall be a string ... in a format blah blah ... We refer to this string as a 'fully qualified logical name'."

I'm referring to the string `"namespaceA::namespaceB"` here, not to the `name` property of any of the `logicalLocationComponent` objects in the array.

```json
"logicalLocations": {
  "namespaceA::namespaceB": [
  ]
}
```

And I'm proposing to refer to a name in this form as a "fully qualified logical name", consistent with the name of the property on the `location` object.